README.md

```diff
@@ -6,114 +6,106 @@
+```
 
-# 4. Usage
-Prepare:
-- A **sample sheet** CSV to specify input/output labels, replicates, etc. (see [example](assets/samplesheet.csv))
-- A **reference FASTA** file for the gene or region of interest
+Secondly, specify the gene or gene region of interest using a reference FASTA file via `--fasta`. Provide the exact codon coordinates using `--reading_frame`.
 
-To execute **nf-core/deepmutscan**, run the basic command:
+Now, you can run the pipeline using:
 
 ```bash
 nextflow run nf-core/deepmutscan \
-  -profile singularity,local \
-  --input ./input.csv \
-  --reading_frame 1-300 \
-  --fasta ./ref.fa \
-  --mutagenesis max_diff_to_wt \
-  --run_seqdepth false \
-  --fitness true \
-  --outdir ./results
+   -profile <docker/singularity/.../institute> \
+   --input ./samplesheet.csv \
+   --fasta ./ref.fa \
+   --reading_frame 1-300 \
+   --outdir ./results
 ```
 
-### Required parameters
-
-| Parameter          | Description                                         |
-|--------------------|-----------------------------------------------------|
-| `--input`          | Path to sample sheet CSV                            |
-| `--outdir`         | Path to output directory                            |
-| `--fasta`          | Reference FASTA file                                |
-| `--reading_frame`  | Start and end nucleotide (e.g. `1-300`)             |
-
-### Optional parameters *(in development)*
+There are several optional parameters, some of which are currently *(in development)*.
 
 | Parameter              | Default     | Description                                     |
 |------------------------|-------------|-------------------------------------------------|
 | `--run_seqdepth`       | `false`     | Estimate sequencing saturation by rarefaction   |
-| `--fitness`            | `false`      | Default fitness inference module                |
+| `--fitness`            | `false`     | Default fitness inference module                |
 | `--dimsum`             | `false`     | Optional fitness inference module *(AMD/x86_64 systems only)* |
-| `--mutagenesis`        | `max_diff_to_wt`   | Deep mutational scanning strategy used *(in development)*         |
-| `--error-estimation`   | `wt_sequencing`     | Error model used to correct 1nt counts *(in development)*         |
-| `--read-align`         | `bwa-mem`   | Read aligner *(in development)*                 |
-
-More options and advanced configuration: [see vignette](link). For further information or help, don't hesitate to get in touch on the [Slack `#deepmutscan` channel](https://nfcore.slack.com/channels/deepmutscan) (you can join with [this invite](https://nf-co.re/join/slack)).
-
----
+| `--mutagenesis`        | `nnk` | Deep mutational scanning strategy used *(in development)* |
+| `--error-estimation`   | `wt_sequencing` | Error model used to correct 1nt counts *(in development)*  |
+| `--read-align`         | `bwa-mem` | Customised read aligner *(in development)* |
```

Comment:
we use the nextflow_schema.json to document parameters (especially addtional), helps keep everything in sync